assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -294,8 +294,10 @@ che.openshift.liveness.probe.delay=300
 che.openshift.liveness.probe.timeout=1
 che.openshift.workspaces.pvc.name=claim-che-workspace
 che.openshift.workspaces.pvc.quantity=10Gi
-che.openshift.workspace.cpu.limit=1
-# Override memory limit used for openshift workspaces. String, e.g. 1300Mi
+# The amount of memory required for a workspace container to run e.g. 512Mi
+che.openshift.workspace.memory.request=NULL
+# The max amount of memory the container can use.
```

Comment:
Since there is some confusion about the format of these properties, maybe we can link to the [OpenShift documentation](https://docs.openshift.org/latest/dev_guide/compute_resources.html#dev-compute-resources) in the description. This does have the problem of having to keep that link up to date though.